lib/samples/show_web_tiled_layer_from_url/show_web_tiled_layer_from_url.dart

```diff
@@ -0,0 +1,95 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import 'package:arcgis_maps/arcgis_maps.dart';
+import 'package:arcgis_maps_sdk_flutter_samples/utils/sample_state_support.dart';
+import 'package:flutter/material.dart';
+
+class ShowWebTiledLayerFromUrl extends StatefulWidget {
+  const ShowWebTiledLayerFromUrl({super.key});
+
+  @override
+  State<ShowWebTiledLayerFromUrl> createState() =>
+      _ShowWebTiledLayerFromUrlState();
+}
+
+class _ShowWebTiledLayerFromUrlState extends State<ShowWebTiledLayerFromUrl>
+    with SampleStateSupport {
+  // Create a controller for the map view.
+  final _mapViewController = ArcGISMapView.createController();
+
+  // A flag for when the map view is ready and controls can be used.
+  var _ready = false;
+
+  // Templated URL to the tile service
+  final String _templateUri =
```

Comment:
Here (and for `_attribution`) we wouldn't specify `String` because the type gets inferred properly. So it'd just be `final _templateUri =`.

I'm also curious if we can use `const` instead of `final`?

Reply:
Actually there's no need for these to class members -- they're only used in one place, so they should just be local variables inside `onMapViewReady`.

Reply:
I've moved them inside the `onMapViewReady()` and set them as `const` also removed the changed their names from `_templateUrl` and `_attribution` to `templateUrl` and `attribution` as they are local variables now.